charts/openmetadata/templates/validate-values.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if and (
```

Comment:
Any specific reason to use the filename with `.yaml` then `.tpl` ?

Reply:
Y, sorry, I think github collapsed all the intermediate section, basically:

https://github.com/open-metadata/openmetadata-helm-charts/pull/93#issuecomment-1336245686